### PR TITLE
Completed Two Column Layout Exercise

### PR DESCRIPTION
**Pull request by maryalenn**
css-two-column-layout

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+p {
+  font: size 14px;
+  color:#585858;
+  line-height: 1.5rem;
+}
+heading {
+  text-align: center;
+}
+#container {
+  width:80%;
```
Comment (BrettAlbright) [inline css-two-column-layout/styles.css]: I would recommend a fixed width for the container, and percentages for the columns and rows.

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  margin: 0 auto;
+}
+
+.col {
```
Comment (BrettAlbright) [inline css-two-column-layout/styles.css:29]: always make sure you explicitly set your column widths. While flex box will automatically generate a size for child elements so they fit evenly in their parent, that cannot be relied upon for stable website layouts

File: css-two-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div id="container">
+    <heading>
+      <h1>Scale, Speed, Agility</h1>
+      <p>We've dedicated thousands of man hours to making healthcare instituitions in the Helium network run 10x more efficiently and deliver an unparalleled quality of care.</p>
+    </heading>
+    <section>
+      <div class="col">
```
Comment (BrettAlbright) [inline css-two-column-layout/index.html:20]: There should be a `row` class which contains the `column` classes, make sure to reference the `layout classes` exercise.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Looks great! A few updates are requested

**Comment by maryalenn** [comment]
Thanks Brett - revisions made
image

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**